package.json

```diff
@@ -46,7 +46,7 @@
     "rimraf": "^3.0.0",
     "source-map-support": "^0.5.19",
     "ts-node": "^10.2.1",
-    "typescript": "4.3.5",
+    "typescript": "4.4.2",
```

Comment:
Right, that should be the authoritative typescript version requirement/definition for all the npm packages we have in our code base (except for those exceptions that _have to_ be exceptions for some reason, and I don't think we have those).